lib/elixir/lib/code.ex

```diff
@@ -28,6 +45,13 @@ defmodule Code do
   Appends a path to the Erlang VM code path.
 
   The path is expanded with `Path.expand/1` before being appended.
+
+  ## Examples
+
+      > Code.append_path(".")
+      > :code.get_path |> List.last |> Kernel.to_string == Path.expand(".")
```

Comment:
I don't think this test makes much purpose. We are checking it is added to the list, we are not asserting anything about its behaviour. To the user, it doesn't tell anything about how it is used and how it affects their code.